recipes/gct/meta.yaml

```diff
@@ -24,9 +24,10 @@ requirements:
     - pkg-config
     - libtool
   host:
-    - openssl =1.0
+    - openssl 1.0.2r
     - zlib
   run:
+    - openssl 1.0.2r
```

Comment:
BTW, `1.0.2r` was just an example. I'm not sure that version is OK for `gct`. Also, do you know it does not work with the latest `1.1.1b`?

Reply:
IIRC `gct` already works with any version of OpenSSL.

The issue is I'm currently build a relatively large chain of dependencies and one of them doesn't (`pygsi`). It's going to take a while until that is patched (or maybe dropped) so I was going to add everything with `OpenSSL 1.0.2` then update in the feedstocks.